.bazelrc

```diff
@@ -23,24 +23,31 @@ build --flag_alias=gcc_version=//:gcc_version
 # Don't include source level debug info on macOS. See
 # https://github.com/urbit/urbit/issues/5561 and
 # https://github.com/urbit/vere/issues/131.
-build:linux --copt='-g'
+build:linux --per_file_copt='//:.*@-g'
```

Comment:
Are we sure `//:.*` is the correct regex? I'm concerned it'll only match files in the top level of the workspace. `//.*\.[ch]` seems like it may be what we want.

Reply:
`//:.*` seems fine. The build has approximately the same elapsed time, so I don't think we're building anything unnecessarily with the simpler regex.

Reply:
My concern isn't that we're rebuilding anything unnecessarily, it's the opposite: I'm concerned we're *not* rebuilding everything we should. The [`--per_file_copt` documentation](https://bazel.build/docs/user-manual#per-file-copt) doesn't make it particularly clear whether `//:.*` will match files in subdirectories of the workspace root or not. I'd think we can answer this definitively using `bazel query` somehow.

Reply:
fyi, tested `//.*\.[ch]` and it doesn't fix the issue.